### PR TITLE
Add support for multiple conditions in @RunIf

### PR DESCRIPTION
**Pull request by theJNOVAK**

### CODE REVIEW

File: tests/org.jboss.reddeer.junit.test/src/org/jboss/reddeer/junit/test/execution/TestMethodShouldRunTest.java [hunk 1]
```diff
@@ -56,4 +57,40 @@ public void testShouldNotRun3() {
 	public void testShouldNotRun4() {
 		fail("Test was not supposed to run because @Ignore annotation is presented and run condition was not met.");
 	}
+	
+	@Test
+	@RunIf(conditionClass={ShouldRun.class, ShouldRun.class}, operation=Operation.AND)
```
Comment (rawagner) [inline tests/org.jboss.reddeer.junit.test/src/org/jboss/reddeer/junit/test/execution/TestMethodShouldRunTest.java]: can you please add test without 'operation' variable defined ? to ensure that default is AND.

Comment (theJNOVAK) [inline tests/org.jboss.reddeer.junit.test/src/org/jboss/reddeer/junit/test/execution/TestMethodShouldRunTest.java]: Added test with no operation specified.

File: plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/runner/RequirementsRunner.java [hunk 2]
```diff
@@ -301,6 +302,28 @@ protected boolean isIgnored(FrameworkMethod child) {
...
 	}
+	
+	/**
+	 * Returns true if operation is AND and all condition classes return true in shouldRun(child).
+	 * Returns true if operation is OR and at least one of condition classes returns true in shouldRun(child).
+	 * Otherwise, returns false.
+	 * 
+	 * @param runIfAnnotation
+	 * @param child
+	 * @return
```
Comment (mlabuda) [inline plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/runner/RequirementsRunner.java]: add description to params and what is returned, if javadoc is provided for private methods, it should still be correct

Comment (theJNOVAK) [inline plugins/org.jboss.reddeer.junit/src/org/jboss/reddeer/junit/internal/runner/RequirementsRunner.java]: Javadoc was copy&pasted from issue. Description added.

**Comment by rawagner** [comment]
testPR

**Pull request merged**